scrapy/core/downloader/handlers/http11.py

```diff
@@ -379,13 +379,14 @@ def _cb_bodyready(self, txresponse, request):
         fail_on_dataloss = request.meta.get('download_fail_on_dataloss', self._fail_on_dataloss)
 
         if maxsize and expected_size > maxsize:
-            error_msg = ("Cancelling download of %(url)s: expected response "
-                         "size (%(size)s) larger than download max size (%(maxsize)s).")
-            error_args = {'url': request.url, 'size': expected_size, 'maxsize': maxsize}
+            warning_msg = ("Expected response size (%(size)s) larger than "
```

Comment:
I think previous error message made more sense, because now we're getting two identical messages for warnsize and maxsize - there is no indication that response is dropped here.